Update .NET SDK mininum VS version (#33388)
dotnet/docs: docs/core/compatibility/sdk/7.0/vs-msbuild-version.md

---
title: "Breaking change: Version requirements for .NET 7 SDK"
description: Learn about the breaking change in the .NET 7 SDK where specific versions of Visual Studio and MSBuild are required.
ms.date: 01/09/2023
---
# Version requirements for .NET 7 SDK

Certain .NET SDK versions require newer versions of Visual Studio and MSBuild.

## Version introduced

.NET SDK 7

## Change description

> The following table shows the minimum version of Visual Studio and MSBuild you'll need to use the .NET 7.0.100 and 7.0.200 SDKs.

Reviewer: ```suggestion
The following table shows the minimum version of Visual Studio and MSBuild you need to use the .NET 7.0.100 or 7.0.200 SDK.
```
